### PR TITLE
app: remove unnecessary code

### PR DESCRIPTION
**Pull request by WuBoBo**

### CODE REVIEW

File: dex/app_test.go [hunk 1]
```diff
@@ -417,17 +417,7 @@ func TestBlockDelivered(t *testing.T) {
...
 		})
 
-	pendingBlock, pendingState := dex.blockchain.GetPending()
-
-	r, ok := dex.app.chainLatestRoot.Load(firstBlocksInfo[0].Block.Position.ChainID)
-	if !ok {
-		t.Errorf("lastest root cache not exist")
-	}
-
-	if *r.(*common.Hash) != pendingBlock.Root() {
```
Comment (aitjcize) [inline dex/app_test.go]: why is this removed?

Comment (WuBoBo) [reply dex/app_test.go]: `r` is taken from chainLatestRoot which has been removed.
